manifests/init.pp

```diff
@@ -58,28 +59,33 @@
           fail("nfs module only supports EL 5, 6 and 7 and operatingsystemmajrelease was detected as <${::operatingsystemmajrelease}>.")
         }
       }
+      $default_nfs_service_required_svcs = undef
     }
     'Solaris': {
       case $::kernelrelease {
         '5.10': {
           $default_nfs_package = [ 'SUNWnfsckr',
-                                    'SUNWnfscr',
-                                    'SUNWnfscu',
-                                    'SUNWnfsskr',
-                                    'SUNWnfssr',
-                                    'SUNWnfssu',
+                                   'SUNWnfscr',
```

Comment:
This breaks puppet-lint

ideal style for this is like

https://github.com/ghoneycutt/puppet-module-nfs/blob/omnios_511/manifests/init.pp#L65